packages/grafana-data/src/types/text.ts

```diff
@@ -0,0 +1,6 @@
+export interface TextMatch {
+  text: string;
+  start: number;
+  length: number;
+  end: number;
+}
```

Comment:
I think this should be added at the top of `grafana-data/src/utils/text.ts`

Most things in `data` are top level items that often stand on their own.  `TextMatch` is really just the response format for `findMatches...`

does not really matter since both locations are exported to the same namespace

Reply:
Good point, will change that